### PR TITLE
Re-enable all CI

### PR DESCRIPTION
**Pull request by jonhoo**

### CODE REVIEW

File: ci/azure-deploy-docs.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
   - template: azure-install-rust.yml
     parameters:
       platform: ${{parameters.name}}
-      rust_version: stable
+      rust: nightly
```
Comment (LucioFranco) [inline ci/azure-deploy-docs.yml]: Pin to 9-10?

Comment (jonhoo) [reply ci/azure-deploy-docs.yml]: Not sure what you mean by 9-10, but I've pinned to 2019-09-24.

Comment (jonhoo) [reply ci/azure-deploy-docs.yml]: Oh, :man_shrugging:, I don't think that's necessary. Do you still want us to pin?

Comment (LucioFranco) [reply ci/azure-deploy-docs.yml]: nah its fine 🚢 it

**Pull request merged**